content/root/quick-start-guide.textile

```diff
@@ -14,6 +14,28 @@ When using Ably in a browser, simply include the Ably Javascript library within
 
 bc[html]. <script src="//cdn.ably.io/lib/ably.min.js"></script>
 
+blang[objc].
+  Add this line to your application's Podfile:
+
+  bc. pod 'Ably', '~> 0.8'
+
+  And install with @pod install@. Then in your files: 
+
+  ```[objc]
+  #import "Ably.h"
+  ```
+
+blang[swift].
+  Add this line to your application's Podfile:
+
+  bc. pod 'Ably', '~> 0.8'
+
+  And install with @pod install@. Then in your files: 
+
+  ```[objc]
```

Comment:
Also, if you read the text at https://github.com/ably/docs/pull/94/files#diff-f9d1ed36577e92a45ffb881113c2205fL13, you'll see that you need to perhaps change that copy to be included in all languages other than the ones you've suggested